sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -20,84 +20,45 @@ autorest
 ### Settings
 
 ```yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/release-cognitiveservices-Language-2022-05-01-preview/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json
```

Comment:
You should actually use a commit so it doesn't change unexpectedly - especially for releases.

Reply:
will do
i'm just using my fork which have some changes i couldn't do using directives in swagger